toolkit/pythonit_toolkit/api/client.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Any, Dict, Optional
+
+import httpx
+import jwt
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+
+INTERNAL_JWT_SECRET = os.getenv("INTERNAL_JWT_SECRET", "a-very-secret-secret")
+
+
+class Client:
+    url = "/internal-api"
+
+    async def execute(
+        self,
+        query: str,
```

Comment:
let's call it document, this client can be used to execute mutations too

```suggestion
        document: str,
```